storm-core/src/native/worker-launcher/impl/configuration.c

```diff
@@ -74,17 +74,17 @@ void free_configurations() {
 static int is_only_root_writable(const char *file) {
   struct stat file_stat;
   if (stat(file, &file_stat) != 0) {
-    fprintf(ERRORFILE, "Can't stat file %s - %s\n", file, strerror(errno));
+    fprintf(ERRORFILE, "ERROR: Can't stat file %s - %s\n", file, strerror(errno));
```

Comment:
nit: consistent indent size of 4 spaces (or two as used here mostly)
nit: stylewise, c code if expressions do not need a != 0 or != NULL part,
           i.e. "if (file_stat.st_uid)" instead of "if (file_stat.st_uid != 0)"